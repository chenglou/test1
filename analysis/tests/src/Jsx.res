module M = {
  @react.component
  let make = (~first, ~fun="", ~second="") => React.string(first ++ fun ++ second)
}

let _ = <M first="abc" />
//       ^def

//^com <M second=fi 

//^com <M second="abc" f

//^com let e = <M 

@react.component
let make = (~first) => React.string(first)

let y = 44

//^com <M prop={A(3)} k
//^com <M prop=A(3) k
