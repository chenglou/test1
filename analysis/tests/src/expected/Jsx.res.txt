Definition tests/src/Jsx.res 5:9
{"uri": "Jsx.res", "range": {"start": {"line": 2, "character": 6}, "end": {"line": 2, "character": 10}}}

Complete tests/src/Jsx.res 7:2
[{
    "label": "key",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "first",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "fun",
    "kind": 4,
    "tags": [],
    "detail": "option<string>",
    "documentation": null
  }]

Complete tests/src/Jsx.res 9:2
[{
    "label": "key",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "first",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "fun",
    "kind": 4,
    "tags": [],
    "detail": "option<string>",
    "documentation": null
  }]

Complete tests/src/Jsx.res 11:2
[{
    "label": "key",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "second",
    "kind": 4,
    "tags": [],
    "detail": "option<string>",
    "documentation": null
  }, {
    "label": "first",
    "kind": 4,
    "tags": [],
    "detail": "string",
    "documentation": null
  }, {
    "label": "fun",
    "kind": 4,
    "tags": [],
    "detail": "option<string>",
    "documentation": null
  }]

